Add more information on DB permissions (#398)
CleverCloud/documentation: layouts/shortcodes/content/managed-services.md

Add-ons are managed services, meaning that users have **standard access** to the database. Some operations like database and user creation, as well as some settings modifications aren't available by default. This ensures optimal performances and security for managed services as configured by Clever Cloud.

If you think your system might require more advanced administrative access, contact Clever Cloud support to explain your use case and we will work with you to find a solution.

Here is the list of actions that you won't be able to perform:

> - Database administration (for example you won't be able to recreate a database after a DROP).

Reviewer: ```suggestion
- Database administration (for example you won't be able to create new databases).
```

Add-ons are managed services, meaning that users have **standard access** to the database. Some operations like database and user creation, as well as some settings modifications aren't available by default. This ensures optimal performances and security for managed services as configured by Clever Cloud.

> If you think your system might require more advanced administrative access, contact Clever Cloud support to explain your use case and we will work with you to find a solution.

Reviewer 2: ```suggestion
If you think your system might require more advanced administrative access, [contact Clever Cloud support](https://console.clever-cloud.com/ticket-center-choice) to explain your use case and we will work with you to find a solution.
```
